### PR TITLE
Feature #56617: Upcoming Screen 

### PR DESCRIPTION
**Pull request by DoQuangKhanh0209**
## [Related Tickets]
- [https://edu-redmine.sun-asterisk.vn/issues/56618](url)
- [https://edu-redmine.sun-asterisk.vn/issues/56620](url)
### [WHAT]
- Create UI Upcoming Screen
- Fetch data Upcoming Screen
#### Screenshots 
simulator_screenshot_4E3E3F60-80EA-4277-AF8B-60EAB5F068A6

simulator_screenshot_5B50A887-4D2C-4CB8-BCDE-0ABD07E4DFCB

### CODE REVIEW

File: Netflix Movies/Netflix Movies/Views/TitleCollectionViewCell.swift [hunk 1]
```diff
@@ -33,7 +32,6 @@ final class TitleCollectionViewCell: UICollectionViewCell, ReusableView {
     }
     
     func configuge(title: Title) {
-        print(title.posterPath)
         if let logoURL = title.posterPath, let url = URL(string: "https://image.tmdb.org/t/p/w500/" + logoURL) {
```
Comment (namtn-2711) [inline Netflix Movies/Netflix Movies/Views/TitleCollectionViewCell.swift]: Bỏ url sang file khác.

File: Netflix Movies/Netflix Movies/Views/TitleTableViewCell.swift [hunk 2]
```diff
@@ -0,0 +1,85 @@
+//
+//  TitleTableViewCell.swift
+//  Netflix Movies
+//
+//  Created by Quang Khánh on 26/11/2022.
+//
+
+import UIKit
+
+class TitleTableViewCell: UITableViewCell {
```
Comment (namtn-2711) [inline Netflix Movies/Netflix Movies/Views/TitleTableViewCell.swift:10]: final

File: Netflix Movies/Netflix Movies/Models/TitleViewModel.swift [hunk 3]
```diff
@@ -0,0 +1,18 @@
+//
+//  TitleViewModel.swift
+//  Netflix Movies
+//
+//  Created by Quang Khánh on 26/11/2022.
+//
+
+import Foundation
+
+struct TitleViewModel: Codable {
```
Comment (namtn-2711) [inline Netflix Movies/Netflix Movies/Models/TitleViewModel.swift:10]: Đặt tên không rõ nghĩa

File: Netflix Movies/Netflix Movies/Views/TitleTableViewCell.swift [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+    }()
+    
+    private let playTitleButton: UIButton = {
+       let button = UIButton()
+        let image = UIImage(systemName: "play.circle", withConfiguration: UIImage.SymbolConfiguration(pointSize: 30))
+        button.setImage(image, for: .normal)
+        button.tintColor = .black
+        button.translatesAutoresizingMaskIntoConstraints = false
+        return button
+    }()
```
Comment (namtn-2711) [inline Netflix Movies/Netflix Movies/Views/TitleTableViewCell.swift:36]: Sao lại khởi tạo bằng code nhỉ

Comment (DoQuangKhanh0209) [reply Netflix Movies/Netflix Movies/Views/TitleTableViewCell.swift:36]: e thấy ít nên e gõ tay ạ

**Pull request closed without merging**